src/modules/infobox/components/FeatureAreaInfo.vue

```diff
@@ -46,7 +46,7 @@ const humanReadableArea = computed(() => {
 
 function roundValueIfGreaterThan(value, threshold, divider) {
     if (value > threshold) {
-        return `${round(value / divider, 2)}`
+        return `${round(value / divider, 3)}`
     }
```

Comment:
Here the value of the number of decimal depends on the divider to avoid having again a 0 value ! Because if a user take a bigger divider you might end up again to a 0 value. Because this function is only used here I suggest to make the rounded decimal in case of threshold reached being a parameter and pass the 3 above in the humanReadableArea function so you have all calcul dependencies at the same place, which will make the code easier to understand and avoid such issue in future.